client/src/components/dashboard/textEditor.js

```diff
@@ -4,17 +4,16 @@ import FroalaEditor from 'react-froala-wysiwyg';
 import jwt from 'jsonwebtoken';
 import { connect } from 'react-redux';
 import toastr from 'toastr';
-// import { bindActionCreators } from 'redux';
 import { createDocument } from '../../actions/docActions';
 
 
-class TextEditor extends React.Component {
+export class TextEditor extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       title: '',
       docContent: '',
-      access: '',
+      access: 'public',
       userId: jwt.decode(localStorage.getItem('JWT')).UserId
```

Comment:
**Code quality - Critical**
Please, your user object should be in the redux store